src/models/network/NetworkAppController.js

```diff
@@ -139,12 +139,14 @@ export default class NetworkAppController extends NetworkBaseController {
       return;
     }
 
-    // TODO: Check that package version matches the requested one
```

Comment:
Note that this TODO is unrelated to the one below. We can convert it to an issue though.

Reply:
Created #175

Reply:
I see it was tackled in another commit, great!